client/src/components/Metrics/MetricsNavigation/MetricsNavigation.tsx

```diff
@@ -12,9 +13,10 @@ import { ActiveSection } from './types';
 const MetricsNavigation = (): ReactElement => {
   const ref = useRef<HTMLDivElement>(null);
   const { i18n, t } = useTranslation('translation', { keyPrefix: 'views.metrics' });
-  const { isDesktop } = useMediaQuery();
+  const { isLargeDesktop } = useMediaQuery();
   const forcedSectionRef = useRef<ActiveSection | null>(null);
   const [activeSection, setActiveSection] = useState<ActiveSection>('epoch');
+  const throttleCallback = useCallback(throttle, []);
```

Comment:
Can you please explain that?

Reply:
throttle method needs to be memoized to use the same instance every render

Reply:
Perhaps I missed something here, but wouldn't wrapping `scrollListener` in throttle directly solve the same problem w/o wrapping `throttle` itself?
We can talk about it tomorrow.